identity/src/main/java/com/android/identity/DataTransportTcp.java

```diff
@@ -349,7 +350,7 @@ DataTransport createDataTransport(
 
         @Override
         Pair<NdefRecord, byte[]> createNdefRecords(List<DataRetrievalAddress> listeningAddresses) {
-            byte[] reference = String.format("%d", DEVICE_RETRIEVAL_METHOD_TYPE)
+            byte[] reference = String.format(Locale.US, "%d", DEVICE_RETRIEVAL_METHOD_TYPE)
```

Comment:
Locale here may also be bad news. @davidz25 would know for sure, but I bet the date format is fixed for the NDEF record id, and changing it for locale will break NFC..